### PR TITLE
Mig17.0 purchase _requisition_send

### PR DESCRIPTION
**Pull request by laszlokr**

### CODE REVIEW

File: purchase_requisition_send/pyproject.toml
Comment (janikvonrotz) [inline purchase_requisition_send/pyproject.toml:1]: Why is this file always showing up?

File: purchase_requisition_order_address/pyproject.toml
Comment (janikvonrotz) [inline purchase_requisition_order_address/pyproject.toml:1]: Same here.

File: purchase_order_address/pyproject.toml
Comment (janikvonrotz) [inline purchase_order_address/pyproject.toml:1]: And here.

**Review by janikvonrotz — CHANGES_REQUESTED** [review]

File: purchase_requisition_send/pyproject.toml
Comment (laszlokr) [reply purchase_requisition_send/pyproject.toml:1]: Apparently it comes from here: 
https://github.com/Mint-System/Odoo-Apps-Purchase-Workflow/blob/ad3a97be7efcb782b25dd58f1bcc101962807570/.pre-commit-config.yaml#L41C2-L44C23

```
Run pre-commit in /home/laszlokr/work/mintsystem/Odoo-Build/addons/purchase_workflow
forbidden files......................................(no files to check)Skipped
en.po files cannot exist.............................(no files to check)Skipped
init pyproject.toml with whool...........................................Failed
- hook id: whool-init
- exit code: 1
- files were modified by this hook

pyproject.toml was generated or modified in purchase_requisition_send, purchase_order_address, purchase_requisition_order_address

```

Comment (janikvonrotz) [reply purchase_requisition_send/pyproject.toml:1]: @laszlokr Can you remove the pre commit config form the 17.0 templates?

Comment (laszlokr) [reply purchase_requisition_send/pyproject.toml:1]: @janikvonrotz I removed the pre-commit config

**Review by janikvonrotz — APPROVED** [review]

**Pull request merged**